app/Controller/CoGroupsController.php

```diff
@@ -227,24 +227,40 @@ function checkWriteDependencies($reqdata, $curdata = null) {
   function checkWriteFollowups($reqdata, $curdata = null, $origdata = null) {
     // Add the co person as owner/member of the new group, but only via HTTP
     
-    if(!$this->request->is('restful') && $this->action == 'add') {
-      $cos = $this->Session->read('Auth.User.cos');
-      $this->redirectTarget = array('action' => 'edit', $this->CoGroup->id);
-      
-      // Member of current CO? (Platform admin wouldn't be)
-      if(isset($cos) && isset($cos[ $this->cur_co['Co']['name'] ]['co_person_id'])) {
-        $a['CoGroupMember'] = array(
-          'co_group_id' => $this->CoGroup->id,
-          'co_person_id' => $this->Session->read('Auth.User.co_person_id'),
-          'owner' => true,
-          'member' => true
-        );
-        
-        if(!$this->CoGroup->CoGroupMember->save($a)) {
-          $this->Flash->set(_txt('er.gr.init'), array('key' => 'information'));
-          return false;
+    if(!$this->request->is('restful')) {
+
+      if($this->action == 'add') {
+        $cos = $this->Session->read('Auth.User.cos');
+        $this->redirectTarget = array('action' => 'edit', $this->CoGroup->id);
+
+        // Member of current CO? (Platform admin wouldn't be)
+        if (isset($cos) && isset($cos[$this->cur_co['Co']['name']]['co_person_id'])) {
```

Comment:
A platform admin _could_ be a member of the current CO, does that matter?

Also, if you test `isset($foo['bar'])` you don't need to also test `isset($foo)`.

Reply:
Shouldn't matter - agree that the comment is inaccurate. Will fix up comment and the test!

Reply:
Update committed as of ee3fa84